### PR TITLE
Update extractjsonfunction.md

### PR DESCRIPTION
**Pull request by urishapira**

### CODE REVIEW

**Comment by PRMerger10** [comment]
@urishapira : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by urishapira** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@urishapira: I'm sorry - only the author of this article, @orspod, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: data-explorer/kusto/query/extractjsonfunction.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ extractjson("$.hosts[1].AvailableMB", EventText, typeof(int))
 
 * *jsonPath*: JsonPath string that defines an accessor into the JSON document.
 * *dataSource*:  A JSON document.
+* *typeLiteral*: An optional type literal (e.g., typeof(int)). If provided, the extracted value is converted to this type.
```
Comment (YaelSchuster) [inline data-explorer/kusto/query/extractjsonfunction.md:30]:
```suggestion
* *typeLiteral*: An optional type literal (for example, typeof(int)). If provided, the extracted value is converted to this type.
```

**Comment by YaelSchuster** [comment]
This has been fixed in a private PR
#please-close

**Pull request closed without merging**